### DISCUSSION TITLE
TMTpro 18plex

### DISCUSSION THREAD

**Discussion by glormph** [OpenMS/OpenMS#6696] [Q&A] [answered]
Hi, our lab is anxiously awaiting a new release of OpenMS, assuming the TMT18plex quant code will be in it (I saw it was merged <3 ).

Since releases of OpenMS are not something we are used to see every month, but the previous one was pretty long ago, I wondered if it was possible to get some kind of non-quantitative estimation here, e.g. "we're [close, far, blocked, ...]". I'm not an OpenMS project manager or funding body, so I understand it's not really any of my business. But TMT18 is getting a rather big thing in our lab.

If the answer is not close, we may go for building from nightly just for the IsobaricAnalyzer, in which case my question becomes "how stable is nightly building?" Thank you :)

**Comment by cbielow** [comment, marked as answer]
we are currently in the process of finalizing the release, and working on the accompanying paper at the same time.
Its kind of interdependent a bit.

I'd hope we can release 3.0 in the coming weeks. There are minor issues/bugs, but these will always be present, at any point in time (the question is just: has someone reported them).
As for stability, you can check the integration test builds at http://cdash.openms.de/index.php?project=OpenMS.
Its pretty stable at the moment. Some quirks here and there, but that's mostly infrastructure-related.

Depending on which platform you need, you can also just try the prebuild binaries (see https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/nightly/ -- sort by date).

**Reply by glormph** [reply]
Thanks very much! Hope the work will be quick for you you guys as well :)
